config/routes.rb

```diff
@@ -3,12 +3,14 @@
 Rails.application.routes.draw do
   post 'auth/login', to: 'authentication#authenticate'
 
-  post 'signup', to: 'users#create'
+  post 'auth/signup', to: 'users#create'
 
   resources :users, only: %i[show update destroy] do
     resources :records
   end
 
+  put '/users/:user_id/records/:id', to: 'records#update_records'
```

Comment:
Let's use a custom name for the route. We can get the user_id from the record, `specialist_id` from the `current_user`. We just need to pass `record_id` and `shedule_date` as parameters as params in our request payload on the front-end.
So we can have
```ruby
put 'claim_user_record', to: 'records#update_records'.
```

Reply:
That makes alot of sense.